bin/dash-network

```diff
@@ -42,4 +42,4 @@ docker run -t --rm \
            -v "$HOME/.aws:/root/.aws" \
            -v "$PUBLIC_KEY_PATH:$PUBLIC_KEY_PATH" \
            -v "$PRIVATE_KEY_PATH:$PRIVATE_KEY_PATH" \
-           dashpay/dash-network-deploy "$@"
+           dashpay/dash-network-deploy:latest-dev "$@"
```

Comment:
I'm afraid we will forget to change it back when we merge v0.23 to master. Any thoughts?

Reply:
I guess we could make it an argument with a blank default value, then pass the arg during CI?